### PR TITLE
CI: Simplify workflow by moving suite specific tests into test script

### PR DESCRIPTION
**Pull request by jurre**
We were setting up a bunch of conditional tests/linting in the workflow
file, but now that we have test scripts I think it's cleaner to run
those inline in the `script/ci-test` script

### CODE REVIEW

**Review by brrygrdn — APPROVED** [review]
👍🏻  for taking this out of the workflow file

File: bundler/script/ci-test [hunk 1]
```diff
@@ -3,3 +3,15 @@
...
 bundle exec rspec spec
+
+[ "$SUITE_NAME" == "bundler1" ] &&\
+  cd /home/dependabot/dependabot-core/bundler/helpers/v1 && \
+  BUNDLER_VERSION=1 bundle install && \
+  BUNDLER_VERSION=1 bundle exec rspec spec &&\
+  cd -
+
+[ "$SUITE_NAME" == "bundler2" ] &&\
+  cd /home/dependabot/dependabot-core/bundler/helpers/v2 && \
```
Comment (feelepxyz) [inline bundler/script/ci-test:14]: Could this work?

```suggestion
  cd helpers/v2 && \
```

File: bundler/script/ci-test [hunk 2]
```diff
@@ -3,3 +3,15 @@
 bundle install
 bundle exec rubocop .
 bundle exec rspec spec
+
+[ "$SUITE_NAME" == "bundler1" ] &&\
+  cd /home/dependabot/dependabot-core/bundler/helpers/v1 && \
```
Comment (feelepxyz) [inline bundler/script/ci-test:8]:
```suggestion
  cd helpers/v1 && \
```

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -98,25 +98,6 @@ jobs:
...
-      - name: Run bundler v1 native helper specs
-        if: matrix.suite.name == 'bundler1'
-        run: |
-          docker run --rm "$CORE_CI_IMAGE" bash -c \
-            "cd /home/dependabot/dependabot-core/bundler/helpers/v1 && BUNDLER_VERSION=1 bundle install && BUNDLER_VERSION=1 bundle exec rspec spec"
-      - name: Run bundler v2 native helper specs
-        if: matrix.suite.name == 'bundler2'
-        run: |
-          docker run --rm "$CORE_CI_IMAGE" bash -c \
-            "cd /home/dependabot/dependabot-core/bundler/helpers/v2 && BUNDLER_VERSION=2 bundle install && BUNDLER_VERSION=2 bundle exec rspec spec"
```
Comment (feelepxyz) [inline .github/workflows/ci.yml:119]: Nice cleanup!

File: bundler/script/ci-test (hunk 1 above)
Comment (jurre) [reply bundler/script/ci-test:14]: Yeah I think so 👍

**Review by feelepxyz — APPROVED** [review]
Sweet! This will simplify this PR https://github.com/dependabot/dependabot-core/pull/3428

File: bundler/script/ci-test [hunk 4]
```diff
@@ -3,3 +3,15 @@
 bundle install
 bundle exec rubocop .
 bundle exec rspec spec
+
+[ "$SUITE_NAME" == "bundler1" ] &&\
```
Comment (feelepxyz) [inline bundler/script/ci-test:7]: Looks like bash is not happy about this?

```
./script/ci-test: 7: [: bundler1: unexpected operator
./script/ci-test: 13: [: bundler1: unexpected operator
```

Comment (feelepxyz) [reply bundler/script/ci-test:7]: wrap in a `if`?

**Pull request merged**